src/feature.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  EvaluationApi,
+  EvaluationBatchRequest,
+  EvaluationBatchRequestFlagTagsOperatorEnum,
+  EvaluationEntity,
+  createConfiguration,
+  ServerConfiguration,
+} from "flagr-client";
+
+export interface FlagCallbacks<T> {
+  [key: string]: (attachment?: JsonObject) => T | Promise<T>;
+}
+
+declare type Json = string | number | boolean | null | JsonArray | JsonObject;
+
+declare type JsonArray = Array<Json>;
+
+declare type JsonObject = {
```

Comment:
`declare` isn't necessary (it'll create global types, which isn't what you want here)